package.json

```diff
@@ -25,6 +25,7 @@
     "chalk": "2.4.1",
     "gulp-filter": "5.1.0",
     "mkdirp": "0.5.1",
+    "npm-api": "^1.0.0",
```

Comment:
shouldn't we use a fixed version instead ?

Reply:
Oh, right! I just fixed it.